README.md

```diff
@@ -3,8 +3,8 @@
 [![Join the chat at https://gitter.im/rubocop-rspec/Lobby](https://badges.gitter.im/rubocop-rspec/Lobby.svg)](https://gitter.im/rubocop-rspec/Lobby)
 [![Gem Version](https://badge.fury.io/rb/rubocop-rspec.svg)](https://rubygems.org/gems/rubocop-rspec)
 [![Build Status](https://secure.travis-ci.org/backus/rubocop-rspec.svg?branch=master)](http://travis-ci.org/backus/rubocop-rspec)
-[![Coverage Status](https://codeclimate.com/github/backus/rubocop-rspec/badges/coverage.svg)](https://codeclimate.com/github/backus/rubocop-rspec/coverage)
-[![Code Climate](https://codeclimate.com/github/backus/rubocop-rspec/badges/gpa.svg)](https://codeclimate.com/github/backus/rubocop-rspec)
+[![Test Coverage](https://api.codeclimate.com/v1/badges/f6254deb61671e357f30/test_coverage)](https://codeclimate.com/github/rubocop-rspec/rubocop-rspec/test_coverage)
```

Comment:
I'm curious, where this hash comes from?

Reply:
Good question. I’ll double check that it isn’t a private api key or something :-)

Reply:
It’s the “badge token” for [our repos on CC](http://developer.codeclimate.com/#get-repository):

```
$ curl -s \
   https://api.codeclimate.com/v1/repos?github_slug=rubocop-rspec/rubocop-rspec | \
   jq '.data[0].attributes.badge_token'
"f6254deb61671e357f30"
```